observability/otel/otel-rn.adoc

```diff
@@ -12,6 +12,95 @@ You can use the {OTELName} xref:../../observability/otel/otel-forwarding-telemet
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="otel_3-6_{context}"]
+== Release notes for {OTELName} 3.6
+
+The {OTELName} 3.6 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.127.0].
+
+[NOTE]
+====
+The {OTELName} 3.6 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.127.0.
+====
+
+[id="otel_3-6_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* https://access.redhat.com/security/cve/CVE-2025-22868[CVE-2025-22868]
+* https://access.redhat.com/security/cve/CVE-2025-22871[CVE-2025-22871]
+
+[id="otel_3-6_technology-preview-features_{context}"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* Tail Sampling Processor
+
+* Cumulative-to-Delta Processor
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel_3-6_new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* The following link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features reach General Availability:
+
+** Kafka Exporter
+
+** Attributes Processor
+
+** Resource Processor
+
+** Prometheus Receiver
+
+* With this update, the OpenTelemetry Collector can read TLS certificates in the `tss2` format according to the TPM Software Stack specification (TSS) 2.0 of the Trusted Platform Module (TPM) 2.0 Library by the Trusted Computing Group (TCG).
+
+* With this update, the Operator automatically upgrades all `OpenTelemetryCollector` custom resources during its startup. The Operator reconciles all managed instances during its startup: if there is an error, the Operator retries the upgrade at exponential backoff. If an upgrade fails, the Operator will retry the upgrade again when it restarts.
+
+////
+[id="otel_3-6_deprecated-functionality_{context}"]
+=== Deprecated functionality
+
+In the {OTELName} 3.6, ???. (link:https://issues.redhat.com/browse/TRACING-????/[TRACING-????])
+////
+
+
+[id="otel_3-6_removal-notice_{context}"]
+=== Removal notice
+
+In the {OTELName} 3.6, the Loki Exporter, which is a temporary Technology Preview feature, is removed. If you currently use the Loki Exporter for Loki 3.0 or later, replace the Loki Exporter with the OTLP HTTP Exporter.
```

Comment:
```suggestion
In the {OTELName} 3.6, the Loki Exporter, which was a temporary Technology Preview feature, is removed. If you currently use the Loki Exporter for Loki 3.0 or later, replace the Loki Exporter with the OTLP HTTP Exporter.
```

Reply:
was it deprecated and then removed, or just removed because never really supported?

Reply:
It was only a temporary Tech Preview feature, which is not supported and thus not subject to the deprecation workflow. And then the Tech Preview feature got dropped (removed).

Thank you. I split the sentence as you suggested, but I kept the future tense as an exception because the future action will take place at a later time.